static/index.html

```diff
@@ -88,14 +92,63 @@ <h1>nobody.live</h1>
           return a.outerHTML;
         }).join(', ');
       }
+      // execute find trainId every 60 seconds
+      var intervalId = window.setInterval(fetchTrainId, 60000)
+      var foundTrainId = false
+      let changingIn = 60
+      function fetchTrainId() {
+        fetch(`${server}/streamtrain`).then(response => response.json())
+          .then(newTrainId => {
+            console.log(newTrainId)
+            trainId = newTrainId;
+            foundTrainId = true;
+            changingIn = 60;
+          }).catch((error) => {
+            console.eror('Error:', error)
+          });
+      }
+      fetchTrainId()
+
+      // join train button show countdown
+      var intervalId = window.setInterval(setChangingIn, 1000)
+      function setChangingIn() {
+        document.getElementById('join-train-button').disabled = false;
+        document.getElementById('join-train-button').innerText = 'join twitch train! new streamer in ' + changingIn + ' seconds';
+        if (changingIn > 0) {
+          changingIn = changingIn - 1;
+        }
+      }
+
+      async function fetchAndRenderStreamTrain() {
```

Comment:
I don't love how much logic is duplicated here. Maybe fetching could be pulled out for more generic UI preparations, or all buttons could be disabled for 4s (I intended that as a low-key rate limiting feature).